polkadot/cli/src/cli.yml

```diff
@@ -104,3 +104,53 @@ subcommands:
             value_name: CHAIN_SPEC
             help: Specify the chain specification (one of dev, local or poc-2)
             takes_value: true
+  - export-blocks:
+      about: Export blocks to a file
+      args:
+          - OUTPUT:
```

Comment:
would be useful to be able to export to stdout (an import from stdin), that way you can just pipe one DB into an other.